packages/ui/src/components/va-file-upload/VaFileUploadGalleryItem/VaFileUploadGalleryItem.vue

```diff
@@ -1,71 +1,65 @@
-
 <template>
-  <div
-    v-if="removed && undo"
-    class="va-file-upload-gallery-item"
-    :class="{ 'va-file-upload-gallery-item--undo': removed }"
-  >
-    <va-file-upload-undo
-      class="va-file-upload-gallery-item--undo"
-      @recover="recoverImage"
-    />
-  </div>
-
-  <div
-    v-else
-    tabindex="0"
+  <va-list-item
     class="va-file-upload-gallery-item"
-    :class="{
-      'file-upload-gallery-item_not-image': !previewImage,
-      'va-file-upload-gallery-item--focused': isFocused,
-    }"
-    @focus="isFocused = true"
-    @blur="isFocused = false"
+    :class="classesComputed"
+    @focus="onFocus"
+    @blur="onBlur"
   >
-    <img
-      v-if="previewImage"
-      :src="previewImage"
-      alt=""
-      class="va-file-upload-gallery-item__image"
-    >
-    <div class="va-file-upload-gallery-item__overlay">
-      <div
```

Comment:
Think it's better to add `file && file.name` checks here and below. Because i've met unpleasant cases with file uploaders.